dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -250,6 +251,14 @@ public class ProcessDefinitionServiceImpl extends BaseServiceImpl implements Pro
     @Autowired
     private ListenerEventAlertManager listenerEventAlertManager;
 
+    private static final int thresholdEntries = 10000;
+    private static final int thresholdSize = 1000000000; // 1 GB
+    private static final double thresholdRatio = 10;
+
+    private final Map<String, DataSource> dataSourceCache = new HashMap<>(1);
```

Comment:
These maps were originally local variables of method `importSqlProcessDefinition`, but they are now private variables of the class, which I think may cause problems

Reply:
+1, this cannot work since ProcessDefinitionServiceImpl is singloten.